webpack.config.js

```diff
@@ -55,11 +59,23 @@ module.exports = {
   },
   plugins: [
     new VueLoaderPlugin(),
-    new MiniCssExtractPlugin({
-      // The files will be placed next to the respective components
-      // i.e.: vue-sidebar.js will be in the same directory as sidebar.css
-      filename: '[name].css',
-      ignoreOrder: false // Maybe we need this if the plugin spits out warnings
-    })
+    new webpack.HotModuleReplacementPlugin(),
+    new webpack.NoEmitOnErrorsPlugin()
```

Comment:
Why is the MiniCssExtractPlugin gone? And if it's not needed anymore, did you remove it from the package.json …?

Furthermore, I don't know if I understand all the changes you've committed to these files, so any explanation would be warmly welcome! :)

Reply:
It didn't seem to do anything, especially since there are no css files but just a few vue files. Also the vue-loader had a `extractCSS` option. But I'm not sure about this...

Reply:
The MiniCssExtractPlugin was necessary (at least when I added the Vue components) because otherweise the vue compiler threw around with errors. If it works now without problems, then we could certainly drop it. Just make sure it doesn't break, then all is good! :) (+ then we can also remove it from the dependencies, obviously …)

Reply:
It works (at least for me) without any problems. I've removed the dependency also from the package.json